src/main/kotlin/com/emberjs/hbs/linter/ember-template-lint/config/TemplateLintUnsavedConfigFileManager.kt

```diff
@@ -0,0 +1,6 @@
+import com.intellij.lang.javascript.linter.LinterUnsavedConfigFileManager
+import com.intellij.openapi.project.Project
+
+class TemplateLintUnsavedConfigFileManager(project: Project) : LinterUnsavedConfigFileManager(project) {
```

Comment:
do we need this custom class if it's the same as `LinterUnsavedConfigFileManager`?

Reply:
Sure I'll remove it. I guess jetbrains does it to avoid plugins using implementation from other packages directly.